lampod/src/ln/inventory_manager.rs

```diff
@@ -38,7 +38,9 @@ impl InventoryHandler for LampoInventoryManager {
                 // we have to put "" in case of alias missing as cln provide us with a random alias.
                 let alias = alias.unwrap_or_default();
                 let (_, height) = self.channel_manager.onchain.backend.get_best_block()?;
-                let blockheight = height.unwrap_or_default();
+                let best_height = height.unwrap_or_default();
```

Comment:
regarding the @Harshit933 review that noted the problem with the `is_syncing` method.

This code is lying, I am not sure why at this point we need a call to `let (_, height) = self.channel_manager.onchain.backend.get_best_block()?;` if we have all the info inside the `self.channel_manager.onchain`. In addition in this case it is not better has `self.onchain`?